dashboard/app/controllers/levels_controller.rb

```diff
@@ -508,6 +508,9 @@ def level_params
       {if_block_options: []},
       {place_block_options: []},
       {play_sound_options: []},
+
+      # Poetry-specific
+      {dropdown_poems: []},
```

Comment:
Nit: `dropdown_poems` feels like slightly off to me as a way to describe what this is. Is it important to indicate here that these are selected with a dropdown field? Would something like `level_poems` or `available_poems` work?

Reply:
I agree, I think `available_poems` is better. I'll update